go.mod

```diff
@@ -47,6 +48,7 @@ require (
 	github.com/bufbuild/buf v1.21.0
 	github.com/golangci/golangci-lint v1.53.3
 	github.com/oasisprotocol/curve25519-voi v0.0.0-20230110094441-db37f07504ce
+	github.com/r2ishiguro/vrf v0.0.0-20180716233122-192de52975eb
```

Comment:
Why did you use replace instead of using `github.com/Finschia/vrf`?

Reply:
`github.com/Finschia/vrf` uses `github.com/r2ishiguro/vrf` inside. So I replaced the package instead of using the fork. Please check the contents of `go.sum`.

Reply:
But I think we will not check the upgrade of `r2ishiguro/vrf` from now, if then, is it better to use our forked ones rather than using replace.

Reply:
`github.com/Finschia/vrf` uses `github.com/r2ishiguro/vrf` inside. So, the fork must be modified, or it would still dependent on `github.com/r2ishiguro/vrf`.

Reply:
We've modified the fork (refer: [Finschia/r2ishiguro_vrf@v0.1.2](https://github.com/Finschia/r2ishiguro_vrf/releases/tag/v0.1.2)). Thank you for your cooperation @zemyblue.